app/assets/javascripts/study_level_activities.js.coffee

```diff
@@ -88,3 +88,7 @@ $ ->
     $.ajax
       type: 'GET'
       url: "/fulfillments/#{fulfillment_id}/edit"
+
+  $(document).on 'click', '.add_fulfillment', ->
+    $('.add_fulfillment').prop('disabled', true)
+    $('.fulfillment-form').submit()
```

Comment:
Do you need this?  Won't it continue down the normal submit path?

Reply:
Yeah, it will still submit the form. But disabling the button prevents the user from creating multiple fulfillments. Before you could rapid click and create like 6 of them.

Reply:
I tried just disabling the button without the $('.fulfillment-form').submit(), but the form would not submit in that case. I needed to do the manual submit after the disable.

Reply:
ok, sounds good.